apps-rendering/src/components/BodyImage/BodyImage.defaults.tsx

```diff
@@ -0,0 +1,123 @@
+// ----- Imports ----- //
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import FigCaption from 'components/FigCaption';
+import { darkModeCss } from '@guardian/common-rendering/src/lib';
+import { ArticleFormat } from '@guardian/libs';
+import { ArticleElementRole } from '@guardian/libs';
+import { Breakpoint } from '@guardian/source-foundations';
+import { from, remSpace } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { none, some, withDefault } from '@guardian/types';
+import Img from 'components/ImgAlt';
+import type { Image } from 'image/image';
+import type { Lightbox } from 'image/lightbox';
+import type { Sizes } from 'image/sizes';
+import type { FC, ReactNode } from 'react';
+
+const width = '100%';
+const phabletWidth = '620px';
+const thumbnailWidth = '8.75rem';
+
+export const getDefaultSizes = (role: ArticleElementRole): Sizes => {
+	switch (role) {
+		case ArticleElementRole.Thumbnail:
+			return {
+				mediaQueries: [],
+				default: thumbnailWidth,
+			};
+		default:
+			return {
+				mediaQueries: [{ breakpoint: 'phablet', size: phabletWidth }],
+				default: width,
+			};
+	}
+};
+
+export const getDefaultThumbnailStyles = (
+	leftColumnBreakpoint: Breakpoint,
+): SerializedStyles => css`
+	float: left;
+	width: ${thumbnailWidth};
+	margin: 0 ${remSpace[3]} 0 0;
+
+	${from[leftColumnBreakpoint]} {
+		position: absolute;
+		transform: translateX(calc(-100% - ${remSpace[4]}));
+	}
+`;
+
+const styles = css`
+	margin: ${remSpace[4]} 0;
+	width: ${width};
+
+	${from.phablet} {
+		width: ${phabletWidth};
+	}
+`;
+
+export const getDefaultStyles = (
+	role: ArticleElementRole,
+	leftColumnBreakpoint: Option<Breakpoint>,
+): SerializedStyles => {
+	switch (role) {
+		case ArticleElementRole.Thumbnail:
+			return getDefaultThumbnailStyles(
+				withDefault<Breakpoint>('leftCol')(leftColumnBreakpoint),
+			);
+		default:
+			return styles;
+	}
+};
+
+export const getDefaultImgStyles = (
+	role: ArticleElementRole,
+	supportsDarkMode: boolean,
+): Option<SerializedStyles> => {
+	switch (role) {
+		case ArticleElementRole.Thumbnail:
+			return some(css`
+				background-color: transparent;
+
+				${darkModeCss(supportsDarkMode)`
+                    background-color: transparent;
+                `}
+			`);
+		default:
+			return none;
+	}
+};
+
+export type BodyImageProps = {
+	image: Image;
+	format: ArticleFormat;
+	supportsDarkMode: boolean;
+	lightbox: Option<Lightbox>;
+	caption: Option<ReactNode>;
+	leftColumnBreakpoint: Option<Breakpoint>;
+};
+
+const DefaultBodyImage: FC<BodyImageProps> = ({
+	image,
+	format,
+	supportsDarkMode,
+	lightbox,
+	caption,
+	leftColumnBreakpoint,
+}) => (
+	<figure css={getDefaultStyles(image.role, leftColumnBreakpoint)}>
```

Comment:
Ideally yes, the issue with this component is that there's two (potentially three) items that need to be styled, the `<figure />` wrapper, the `<img />` and the `<FigCaption />`. So it's not as simple as adding the two props you mentioned and passing a `css` prop to the component, it will have to be something more like:
```
figureStyles: SerializedStyles;
imgStyles: SerializedStyles;
captionStyles: SerializedStyles; 
```